### PR TITLE
Feature/pelagos 4536 add custom template for hri

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: assets/js/vue/components/search/Facet.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
                 <div class="input-group pb-3" v-show="['researchGroup', 'projectDirector'].includes(facetName.queryParam)">
                     <input class="form-control" placeholder="Search" type="text" v-model="facetSearch">
                     <div class="input-group-append">
-                        <button class="btn btn-primary" type="button">
+                        <button class="btn btn-alternate" type="button">
```
Comment (mickel1138) [inline assets/js/vue/components/search/Facet.vue:13]: This will also change the color for the other templates, are "we" ok with that?

**Review by mickel1138 — APPROVED** [review]
LGTM!

File: assets/js/vue/components/search/Facet.vue (hunk 1 above)
Comment (mickel1138) [reply assets/js/vue/components/search/Facet.vue:13]: It appears they are the same colors, so should be ok.

Comment (praneethpr) [reply assets/js/vue/components/search/Facet.vue:13]: Yes, it should be because for other templates we have primary and alternate as the same color

**Pull request merged**